### ISSUE TITLE
v3.2.3 fails to install

### ISSUE THREAD

**Issue by cqc-alec** [HQSquantumsimulations/PyQuEST-cffi#26]
`pip install pyquest-cffi==3.2.3` fails (on Linux) with the message:

```
    CMake suite maintained and supported by Kitware (kitware.com/cmake).
    __file__ /tmp/pip-install-zdnpfpa9/pyquest-cffi/setup.py
    src_path /tmp/pip-install-zdnpfpa9/pyquest-cffi
    CMake Error: The current CMakeCache.txt directory /tmp/pip-install-zdnpfpa9/pyquest-cffi/pyquest_cffi/questlib/CMakeCache.txt is different than the directory /home/nvogt/vanilla/HQS_pyquest/pyquest_cffi/questlib where CMakeCache.txt was created. This may result in binaries being created in the wrong place. If you are not sure, reedit the CMakeCache.txt
    CMake Error: The source "/tmp/pip-install-zdnpfpa9/pyquest-cffi/QuEST/QuEST/CMakeLists.txt" does not match the source "/home/nvogt/vanilla/HQS_pyquest/QuEST/QuEST/CMakeLists.txt" used to generate cache.  Re-run cmake with a different source directory.
```

v3.2.2 installs fine.

**Comment by nfwvogt** [comment]
Thank you for reporting!
CMake failed due to old CMake cache when building the sdist distribution.
This should be fixed now in 3.2.3.1

**Comment by cqc-alec** [comment]
Yes, fixed!
Thanks!
